mautrix/bridge/commands/admin.py

```diff
@@ -29,3 +29,92 @@ async def set_power_level(evt: CommandEvent) -> EventID:
     except (MatrixRequestError, IntentError):
         evt.log.exception("Failed to update power levels")
         return await evt.reply("Failed to update power levels (see logs for more details)")
+
+@command_handler(needs_admin=True, needs_auth=False, name="set-avatar",
+                 help_section=SECTION_ADMIN, help_args="<_mxc_uri_> [_mxid_]",
+                 help_text="Sets an avatar for a ghost user.")
+async def set_ghost_avatar(evt: CommandEvent) -> EventID:
+    try:
+        mxc_uri = evt.args[0]
+    except (KeyError, IndexError):
+        return await evt.reply("**Usage:** `$cmdprefix+sp set-avatar <mxc_uri> [mxid]`")
+    if not mxc_uri.startswith("mxc://"):
+        return await evt.reply("The URI has to start with mxc://.")
+    if len(evt.args) > 1:
+        puppet = await evt.processor.bridge.get_puppet(evt.args[1])
```

Comment:
parsing mention pills would be a nice extra, but not particularly important, so a comment like this is enough

```suggestion
        # TODO support parsing mention pills instead of requiring a plaintext mxid
        puppet = await evt.processor.bridge.get_puppet(evt.args[1])
```

Reply:
You want that in all commands or just on top of the first one?

Reply:
This is probably enough for just the first one